python/GafferOFXTest/PluginCacheTest.py

```
@@ -0,0 +1,53 @@
+
+import unittest
+
+import imath
+
+import IECore
+import GafferTest
+import GafferOFX
+
+class PluginCacheTest( GafferTest.TestCase ) :
+	@unittest.skip("")
+	def testFindPlugins(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `testFindPlugins`

Add return type annotation: `None`

(ANN201)